https://github.com/OpenSMTPD/OpenSMTPD/pull/1199
https://bugs.gentoo.org/720782

From 92ada4471602fc737113b8dfe1b9b8e8e0aab7e0 Mon Sep 17 00:00:00 2001
From: orbea <orbea@riseup.net>
Date: Tue, 25 Apr 2023 09:13:51 -0700
Subject: [PATCH] configure: Use AM_PROG_AR

Automake provides AM_PROG_AR as a standard way of finding ar(1).

Reference: https://www.gnu.org/software/automake/manual/html_node/Public-Macros.html

Closes: https://github.com/OpenSMTPD/OpenSMTPD/pull/1177
---
 configure.ac | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/configure.ac b/configure.ac
index 00450485..5b12b67e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -67,7 +67,7 @@ AC_C_BIGENDIAN
 AC_PROG_CPP
 AC_PROG_INSTALL
 AC_PROG_LIBTOOL
-AC_PATH_PROG([AR], [ar])
+AM_PROG_AR
 AC_PATH_PROG([CAT], [cat])
 AC_PATH_PROG([CHMOD], [chmod])
 AC_PATH_PROG([CHOWN], [chown])
@@ -84,11 +84,6 @@ AC_PROG_YACC
 
 AC_SUBST([ZCAT])
 
-
-if test -z "$AR"; then
-	AC_MSG_ERROR([*** 'ar' missing, please install or fix your \$PATH ***])
-fi
-
 if test -z "$LD"; then
 	LD=$CC
 fi
